Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -273,7 +299,7 @@ public async Task<ActionResult> SyncForUser(int id)
 
     [Authorize]
     [HttpGet("libraries")]
-    public ActionResult<Directory[]> GetLibraries()
+    public ActionResult<Shoko.Models.Plex.Libraries.Directory[]> GetLibraries()
```

Comment:
```suggestion
    public ActionResult<Directory[]> GetLibraries()
```

Use an import, or if there is a clash, an alias.